### ISSUE TITLE
Mounting on horses and modded mounts not working properly

### ISSUE THREAD

**Issue by MikeDaBird** [jrbudda/minecrift#268]
It seems that when I ride a horse, the camera is way too low (the head is almost touching the saddle). It looks like it happens also with mods such as the dragon mount mod. Minecarts seems to work fine.

The effect can easily be reproduced in both standing and sitting modes.
This bug is both in single player and multiplayer AND both with forge and without it.
![horse](https://cloud.githubusercontent.com/assets/24901068/21734385/1d064d9c-d431-11e6-921c-e357f0cc686d.gif)
![dragon](https://cloud.githubusercontent.com/assets/24901068/21734386/1d0753b8-d431-11e6-8b85-a09902614b18.gif)
(I included the dragon in the post because the bug is more visible there than on the horse)

**Comment by jrbudda** [comment]
Could you try with the latest release?

**Comment by MikeDaBird** [comment]
Yes, the problem is also in the latest release (for 1.11). I just tested it right now.

**Comment by jrbudda** [comment]
what really? even 1.11.2? I know I fixed it 1.11.2 where you sit up where you should with vanilla horses.

**Comment by MikeDaBird** [comment]
My bad. I didn't try 1.11.2 and you are right, you did fix it. It feels just a bit too high tho (like if you were standing on the horse) and moving your head left and right (not turning your head) doesn't move the head in-game.

Still, the bug is in 1.11 (not 1.11.2) and 1.7.10 (forge and non-forge).

**Comment by MikeDaBird** [comment]
Thanks for the 36r1 update for 1.7.10, the problem is resolved now for the horses. Still didn't solve the bug on the dragon, but at least vanilla mounts work properly.
